pettingzoo/utils/save_observation.py

```diff
@@ -1,10 +1,14 @@
+from __future__ import annotations
+
 import os
 
 import gymnasium.spaces
 import numpy as np
 
+from pettingzoo.utils.env import AECEnv, AgentID, ParallelEnv
+
 
-def _check_observation_saveable(env, agent):
+def _check_observation_saveable(env: AECEnv | ParallelEnv, agent: AgentID):
```

Comment:
These should have -> None to match all the other parts where you do return None (see https://stackoverflow.com/questions/48223558/typehints-none-or-leave-blank)